runbotics/runbotics-orchestrator-ui/src/src-app/views/users/DeleteUserDialog.tsx

```diff
@@ -50,7 +51,8 @@ const DeleteUserDialog: FC<DeleteUserDialogProps> = ({
     const dispatch = useDispatch();
 
     const { userDelete } = useSelector(usersSelector);
-    const { refreshSearch } = useUserSearch(false);
+    const { refreshSearch: refreshSearchNotActivated } = useUserSearch(false);
+    const { refreshSearch: refreshSearchActivated } = useUserSearch(true);
```

Comment:
what boolean parameter does?

Reply:
It gives information to the function to know if it is using search for activated users or for not activated (mainly for dispatch actions)